lib/Capabilities.php

```diff
@@ -54,59 +54,58 @@ public function getCapabilities(): array {
 			return [];
 		}
 
-		$maxChatLength = 1000;
-		if (version_compare($this->serverConfig->getSystemValueString('version', '0.0.0'), '16.0.2', '>=')) {
-			$maxChatLength = ChatManager::MAX_CHAT_LENGTH;
-		}
-
-		$attachments = [
-			'allowed' => $user instanceof IUser,
+		$capabilities = [
+			'features' => [
+				'audio',
+				'video',
+				'chat-v2',
+				'conversation-v2',
+				'guest-signaling',
+				'empty-group-room',
+				'guest-display-names',
+				'multi-room-users',
+				'favorites',
+				'last-room-activity',
+				'no-ping',
+				'system-messages',
+				'mention-flag',
+				'in-call-flags',
+				'notification-levels',
+				'invite-groups-and-mails',
+				'locked-one-to-one-rooms',
+				'read-only-rooms',
+				'chat-read-marker',
+				'webinary-lobby',
+				'start-call-flag',
+				'chat-replies',
+				'circles-support',
+				'force-mute',
+				'chat-reference-id',
```

Comment:
chat-reference-id should be removed from the default capability list

Reply:
🤦